### DISCUSSION TITLE
Width and borders

### DISCUSSION THREAD

**Discussion by gldtn** [simonmclean/tryptic#50] [Q&A]
Can I change border chars? 

I tried doing something similar to what telescope allows me to do, but that didn't work 🤣 
```lua
borderchars = { " ", "", "", "▌", "▌", "", "", "▌" },
```
Which returns something like this width a nice divider separating the preview column:
![tele](https://github.com/simonmclean/triptych.nvim/assets/981383/1c55ce92-f6e3-49d1-ac2e-566520bb1d38)

Also can you adjust the width, height of the popup window other than adjusting the responsive column widths?

Thank you!

**Comment by simonmclean** [comment]
@gldtn Not at the moment, but I could add these as config options. Can you describe how you'd like to change the width and height? (in terms of what the Triptych view should look like, not the config)

**Reply by gldtn** [reply]
Hey Simon, I guess being able to control the width/height overall of the popup by setting a percentage  number, like you can width telescope, that would be nice. Some people like it so it takes the entire screen of neovim where others prefer to make a bit smaller.

So instead of this:
![image](https://github.com/simonmclean/triptych.nvim/assets/981383/4bc6ae22-736e-4997-a564-942959cee2f6)

It would look more similar to this:
![image](https://github.com/simonmclean/triptych.nvim/assets/981383/7bfb2681-6171-4d2a-b9a7-cf987033a9bb)

On the second section, I modified the max_total_width to 180 and max_height to 65 just as an example!

Thanks for considering it.

**Reply by simonmclean** [reply]
@gldtn I've added a border option to the config. Height and width will be next on the list

**Reply by gldtn** [reply]
Thank you for adding this 😀

**Reply by simonmclean** [reply]
@gldtn Sorry for the delay. Took a break from the project, but I'm back now 💪 Height, width and margin options have been added. Let me know if this suits your needs
